src/Adapter/Http.php

```diff
@@ -483,6 +503,10 @@ protected function _basicAuth($header)
 
         // Decode the Authorization header
         $auth = substr($header, strlen('Basic '));
+        if ($auth === false) {
+            return $this->challengeClient();
```

Comment:
Checking if a test exists is needed here

Reply:
Path was not covered yet. Test added.